client/me/security-2fa-initial-setup/style.scss

```diff
@@ -0,0 +1,11 @@
+.security-2fa-initial-setup__label span {
+    display: inline;
+    margin: 0 0 0 5px;
+    padding: 0;
+    font-size: 1rem; /* stylelint-disable-line */
```

Comment:
Or we could just use `$font-body` instead of `1rem`? That way we wouldn't need to disable the linter.